analyze.py

```diff
@@ -51,14 +61,29 @@ def O365_analyze(self, responses):
         for line in responses:
             results.append(line[0])
         success_indicies = list(filter(lambda x: results[x] == 'Success', range(len(results))))
+
         # print out logins with outlying response lengths
         if len(success_indicies) > 0:
             self.colors.color_print('[+] Identified potential sussessful logins!\n', self.colors.green)
             table = Texttable(0)
             table.header(['Username', 'Password', 'Message'])
             for x in success_indicies:
                 table.add_row([responses[x][2], responses[x][3], responses[x][1]])
+
+            # Calling notifications if specified
+            if self.notify == 'slack':
+                slack(self.webhook, self.host)
+            elif self.notify == 'teams':
+                teams(self.webhook, self.host)
+            elif self.notify == 'discord':
+                teams(self.webhook, self.host)
+
             print(table.draw())
+
+            # Returning true to indicate a successfully guessed credential
+            return True
+
+
         else:
             self.colors.color_print('[-] No successful Office365 logins', self.colors.red)
 
```

Comment:
return false for `check_sleep()` call in `spraycharles.py`

```suggestion
            return False
```